use Core.print

error ErrInline:
	Throwing;

def will_throw():
	throw ErrInline.Throwing("Wahu");

def main():
	will_throw() catch:
		ErrInline(b): print("is inline thrown\n");
		anyerror(a): print("is anyerror\n");
